src/components/base/toast.vue

```diff
@@ -110,12 +120,32 @@ export enum ToastType {
 
 <script setup lang="ts">
 import { ref, type PropType } from 'vue';
+import { removeToast, type Toast } from '@stores/toasts';
 
 const showToast = ref(true);
 
+const isMouseOver = ref(false);
+
+let value = ref(100);
```

Comment:
Use const and also maybe rename to progressValue or something more descriptive

```suggestion
const value = ref(100);
```